docs/products/kafka/karapace/howto/manage-karapace-kafka-rest-authorization.rst

```diff
@@ -1,5 +1,5 @@
-Manage Karapace schema registry authorization
-=============================================
+Manage Karapace Kafka REST proxy schema registry authorization
```

Comment:
should this be Apache Kafka REST...

Reply:
That is right. Much of the content is still a work in progress. I will fix this one :)